src/main/java/io/ebean/config/ServerConfig.java

```diff
@@ -518,6 +524,10 @@
    */
   private boolean idGeneratorAutomatic = true;
 
+  private LoadErrorHandler loadErrorHandler = (bean, prop, fullName, e) -> {
```

Comment:
I think this should instead be on the query itself rather than global.  query().setAllowLoadErrors().where()....findOne()

Reply:
Does it also trigger on a lazy load then? (I think in my use case it could be OK to have that on the query - have to check)